config/settings/base.py

```diff
@@ -384,7 +384,12 @@
 UMB_BREW_MONITOR_ENABLED = strtobool(os.getenv("CORGI_UMB_BREW_MONITOR_ENABLED", "true"))
 
 # Set to True to turn on loading of community products from product-definitions.
-COMMUNITY_MODE_ENABLED = strtobool(os.getenv("CORGI_COMMUNITY_MODE_ENABLED", "false"))
+if CORGI_DOMAIN.endswith(".fedoraproject.org"):
+    COMMUNITY_MODE_ENABLED = True
+    CORGI_DOMAIN_BASE = ".fedoraproject.org"
+else:
+    COMMUNITY_MODE_ENABLED = False
+    CORGI_DOMAIN_BASE = ".redhat.com"
```

Comment:
Unfortunately we can't set the COMMUNITY_MODE_ENABLED flag in this check because stage still uses an internal URL. I didn't request an external cname and SSL cert for community stage at this time. We could optionally request that and have this change work as written. But for now, I think we should change it to just set the CORGI_DOMAIN_BASE setting.

```suggestion
COMMUNITY_MODE_ENABLED = strtobool(os.getenv("CORGI_COMMUNITY_MODE_ENABLED", "false"))

if CORGI_DOMAIN.endwith(".fedoraproject.org"):
    CORGI_DOMAIN_BASE = ".fedoraproject.org"
else:
    CORGI_DOMAIN_BASE = ".redhat.com"
    
```

Reply:
The API is deployed on the internal CORGI_DOMAIN for community-stage instance, so I'm going to merge this updated as I suggested here. We can always migrate to an external host for community-stage if required later.